satellite/admin-back-office-v1.md

```diff
@@ -74,42 +74,44 @@ with or without any relation with Storj will be able to use it.
 
 ##### Operations
 
-1. Account:
+1 Account:
   1. View information: ID, email, full name, paid/free tier, suspended?, user agents, Multi-factor
-     authentication enabled/disabled, data placement restriction (geo fence & SOC2), list of projects,
+     authentication enabled/disabled, data placement restriction (geo-fence & SOC2), list of projects,
      limits (download, storage, number of segments, number of projects allowed).
   2. Change email address.
   3. Disable Multi-factor authentication.
-  5. Set limits (download, storage, number of segments, number of projects allowed).
-  6. Suspend "disable upload/download":
+  4. Set limits (download, storage, number of segments, number of projects allowed).
```

Comment:
Oh, one common way to avoid having to renumber markdown numeric lists is to use `1.` everywhere :). But, actual values are generally better.